src/eth/EthereumCdpService.js

```diff
@@ -227,7 +227,9 @@ export default class EthereumCdpService extends PrivateService {
     // from the non-constant function `tab`
     const tub = this._smartContract().getWeb3ContractByName(contracts.SAI_TUB);
     const tab = await new Promise((resolve, reject) =>
-      tub.tab.call(hexCdpId, (err, val) => (err ? reject(err) : resolve(val)))
+      tub.methods
+        .tab(hexCdpId)
+        .call({}, (err, val) => (err ? reject(err) : resolve(val)))
```

Comment:
The new web3 call methods are somewhat clunky in comparison to what the previous version was. 
Instead of 
```
contract.methods.method(args).call(options, cb)
```
It could be more legible to restructure to
```
contract.method(args, options, cb)
```